corehq/apps/callcenter/tasks.py

```diff
@@ -58,12 +69,18 @@ def sync_usercases_if_applicable(user, spawn_task):
         return
     if (user.project.call_center_config.enabled or user.project.usercase_enabled):
         if spawn_task:
-            sync_usercases_task.delay(user._id)
+            sync_usercases_task.delay(user._id, user.project)
         else:
-            sync_usercases_task(user._id)
+            sync_usercases_task(user._id, user.project)
+
+
+@task(serializer='pickle', queue='background_queue')
+def sync_webuser_usercases_task(user, domain_obj):
+    sync_usercases(user, domain_obj)
+    return None
 
 
 @task(serializer='pickle', queue='background_queue')
```

Comment:
once domain name is being passed in instead of the whole domain:
```suggestion
@task(queue='background_queue')
```